src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,224 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    private static class Node<T> {
+        private T item;
+        private Node<T> next;
+        private Node<T> previous;
+
+        private Node(T element) {
```

Comment:
Why we need 2 constructors here? Could we do the same only with second one?

Reply:
Fully agree. I deleted that.